schemas/0.1.0/property-template.json

```diff
@@ -77,10 +85,10 @@
           "items": {
             "type": "string"
           },
+          "minItems": 1,
           "example": [
-            "['profile:bf2:Agent:Person', 'profile:bf2:Identifiers:Barcode', 'profile:bflc:Agents:PrimaryContribution']"
-          ],
-          "minItems": 1
+            "['sinopia:resourceTemplate:bf2:Agent:Person', 'sinopia:resourceTemplate:bf2:Identifiers:Barcode', 'sinopia:resourceTemplate:bflc:Agents:PrimaryContribution']"
+          ]
```

Comment:
I figure better to refer to less confusing resource template ids!

Reply:
:+1: